frontend/talentsearch/src/js/components/reviewMyApplication/ReviewMyApplicationPage.tsx

```diff
@@ -0,0 +1,258 @@
+              poolAdvertisement.id,
+            )}?application=${poolCandidateId}`,
+          },
+          skillsExperience: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.skillsAndExperiences(
+              poolAdvertisement.id,
+            )}?application=${poolCandidateId}`,
+            override: (
+              <>
+                {missingSkills && (
+                  <div data-h2-margin="base(x1, 0)">
+                    <MissingSkills
+                      addedSkills={
+                        hasExperiences
+                          ? flattenExperienceSkills(experiences)
+                          : []
+                      }
+                      requiredSkills={missingSkills.requiredSkills}
+                      optionalSkills={missingSkills.optionalSkills}
+                    />
+                  </div>
+                )}
+                {!hasExperiences ? (
+                  <div
+                    data-h2-radius="base(s)"
+                    data-h2-background-color="base(light.dt-gray)"
+                    data-h2-padding="base(x1)"
+                  >
+                    <p data-h2-font-style="base(italic)">
+                      {intl.formatMessage({
+                        defaultMessage:
+                          "There are no experiences on your profile yet. You can add some using the preceding buttons.",
+                        description:
+                          "Message to user when no experiences have been attached to profile.",
+                      })}
+                    </p>
+                  </div>
+                ) : (
+                  <ExperienceSection experiences={experiences} />
+                )}
+                <div
+                  data-h2-display="base(flex)"
+                  data-h2-padding="base(x2, 0, 0, 0)"
+                  style={{ gap: "1rem" }}
+                >
+                  <Link
+                    href={directIntakePaths.signAndSubmit(applicant.id)}
+                    color="cta"
+                    mode="solid"
+                    type="button"
+                    disabled={isProfileComplete}
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Continue to step 2",
+                      description:
+                        "Button message on footer of review my application page.",
+                    })}
+                  </Link>
+                  <Link
+                    href={directIntakePaths.applications(applicant.id)}
+                    color="black"
+                    mode="inline"
+                    type="button"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Save and go back to my applications",
+                      description:
+                        "Button message on footer of review my application page.",
+                    })}
+                  </Link>
```

Comment:
this says "save and go back", but it doesn't appear to do any saving? should it?
I think all it should do is "go back"

Reply:
That's what is says on the wireframe so I went with that. I can ask Jerbo and see what he says.